test/new-e2e/tests/updater/linux_test.go

```diff
@@ -106,8 +103,9 @@ func (v *vmUpdaterSuite) TestAgentUnitsLoaded() {
 }
 
 func (v *vmUpdaterSuite) TestExperimentCrash() {
-	host := v.Env().RemoteHost
 	t := v.T()
+	t.Skip()
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** Why are we skipping this test?‏

Reply:
it's currently broken by https://github.com/DataDog/datadog-agent/pull/24507, which installs a faulty oci agent (systemd units always crash) Arthur is fixing it and on the mean time I'm skipping the test